source/neuropod/internal/backend_registration.cc

```diff
@@ -63,12 +63,16 @@ const std::vector<BackendLoadSpec> default_backend_for_type = {
     {"torchscript", "1.2.0", "libneuropod_torchscript_backend.so"},
     {"torchscript", "1.3.0", "libneuropod_torchscript_backend.so"},
     {"torchscript", "1.4.0", "libneuropod_torchscript_backend.so"},
+    {"torchscript", "1.5.0", "libneuropod_torchscript_backend.so"},
```

Comment:
I remember I was able to build our service with Torch 1.5 and tests passed successfully. I guess it because our model doesn't specify version but platform only and this is why correct backend was registered anyway.